config.json

```diff
@@ -67,6 +67,7 @@
         "enabled": true,
         "channelName": "reports",
         "staffRoleName": "Staff"
+        
```

Comment:
random space?!

Reply:
had to edit the config file to put the user id's back, oops thats gonna get removed